app/src/main/java/xyz/whereuat/whereuat/KeyLocDialogFragment.java

```diff
@@ -79,6 +86,18 @@ public void onFocusChange(View v, boolean hasFocus) {
     }
 
     private void addKeyLoc(Context context, String name, Location loc) {
-        new KeyLocation(name, loc.getLatitude(), loc.getLongitude()).dbInsert(context);
+        double latitude = loc.getLatitude();
```

Comment:
So you're saying just go over the line length here? Or break the function into two lines?